### ISSUE TITLE
Latest build of CentOS7 AMI cannot login via SSH

### ISSUE THREAD

**Issue by syuu1228** [scylladb/scylla-machine-image#196]
On latest build of CentOS7 AMI could not login via SSH, and getting dracut error on system log.
I got this when I build latest next branch on local environment, and also on jenkins:
https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/ami/717/
Looks like some failure happens on initramfs stage, can be related with kernel-ml package.

```
[    3.337884] FDC 0 is a S82078B
         Starting dracut initqueue hook...
[    3.352106] ena 0000:00:03.0: ENA device version: 0.10
[    3.357469] ena 0000:00:03.0: ENA controller version: 0.0.1 implementation version 1
[    3.365367] scsi host0: ata_piix
[    3.365483] ena 0000:00:03.0: LLQ is not supported Fallback to host mode policy.
[    3.369942] scsi host1: ata_piix
[    3.379698] ata1: PATA max MWDMA2 cmd 0x1f0 ctl 0x3f6 bmdma 0xc100 irq 14
[    3.379973] ena 0000:00:03.0: Elastic Network Adapter (ENA) found at mem 83000000, mac addr 02:d7:dd:bb:92:cb
[    3.385039] ata2: PATA max MWDMA2 cmd 0x170 ctl 0x376 bmdma 0xc108 irq 15
[    3.415675] nvme nvme0: pci function 0000:00:1e.0
[    3.440559] nvme nvme0: 15/0/0 default/read/poll queues
[  128.522397] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  129.047311] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  129.563310] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  130.081302] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  130.597376] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  131.116381] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  131.633322] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  132.151325] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  132.669326] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  133.187363] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  133.705461] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  134.224333] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  134.741317] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  135.259324] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  135.776318] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  136.294375] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  136.811318] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  137.328316] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  137.844321] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  138.362410] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  138.879405] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  139.396327] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  139.912400] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  140.429337] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  140.946331] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  141.464309] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  141.980309] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  142.497335] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  143.014388] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  143.531419] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  144.048326] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  144.566368] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  145.083395] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  145.600404] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  146.117325] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  146.635320] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  147.151312] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  147.669324] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  148.185317] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  148.703379] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  149.220329] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  149.738393] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  150.256370] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  150.775383] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  151.292379] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  151.811448] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  152.328489] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  152.845381] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  153.362377] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  153.879392] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  154.396306] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  154.914320] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  155.430353] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  155.947311] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  156.464301] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  156.982302] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  157.498291] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  158.016359] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  158.533330] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  159.051366] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  159.568370] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  160.085372] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  160.601516] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  161.120366] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  161.637385] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  162.156384] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  162.673315] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  163.191324] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  163.708308] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  164.226380] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  164.743308] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  165.261398] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  165.778330] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  166.296307] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  166.815460] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  167.334313] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  167.850332] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  168.368312] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  168.884300] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  169.402376] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  169.919378] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  170.436307] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  170.952315] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  171.470298] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  171.987338] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  172.504369] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  173.022322] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  173.539377] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  174.056312] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  174.575321] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  175.094431] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  175.613302] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  176.132410] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  176.651296] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  177.167309] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  177.685312] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  178.202315] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  178.720388] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  179.237394] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  179.754330] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  180.270311] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  180.788372] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
[  181.305391] dracut-initqueue[353]: Warning: dracut-initqueue timeout - starting timeout scripts
```

**Comment by syuu1228** [comment]
To make sure it related with kernel-ml package, I tried to generate normal kernel AMI image by deleting `yum -y --enablerepo=elrepo-kernel install kernel-ml` line from scylla_install_image, it just works.

**Comment by syuu1228** [comment]
I guessed latest version of  kernel-ml package causes the error, because our AMI build worked exactly same code in few days ago.
So I tested to build an AMI with older kernel-ml package, like this:
```
diff --git a/packer/scylla_install_image b/packer/scylla_install_image
index 677c53f..c421118 100755
--- a/packer/scylla_install_image
+++ b/packer/scylla_install_image
@@ -221,9 +221,7 @@ if __name__ == '__main__':
         # So we have to use different kernel here.
         if platform.machine() == 'x86_64':
             run('yum -y install grubby')
-            run('rpm --import https://www.elrepo.org/RPM-GPG-KEY-elrepo.org')
-            run('rpm -Uvh http://www.elrepo.org/elrepo-release-7.0-3.el7.elrepo.noarch.rpm')
-            run('yum -y --enablerepo=elrepo-kernel install kernel-ml')
+            run('yum -y install https://elrepo.org/linux/kernel/el7/x86_64/RPMS/kernel-ml-5.13.13-1.el7.elrepo.x86_64.rpm')
             mlkver = get_kver('/boot/vmlinuz-*el7.elrepo.x86_64')
             run('grubby --grub2 --set-default /boot/vmlinuz-{mlkver}'.format(mlkver=mlkver))
             with open('/etc/yum.conf', 'a') as f:
```
It just worked, so I think the error caused by latest version of kernel-ml package, kernel-ml-5.14.0-1.el7.elrepo.x86_64.

**Comment by syuu1228** [comment]
Maybe we need to similar with 708443337782b42f7c38dbe9c383a7d68d731f78 until we found a solution, or fixed the issue on next version of the package.

**Comment by syuu1228** [comment]
I could reproduce this also on CentOS7 AMI (NOT Scylla AMI, CentOS official one).
So it is highly likely not our AMI problem but the kernel package problem.
```
 4.119321] nvme nvme1: 31/0/0 default/read/poll queues
[  129.059941] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  129.586841] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  130.104848] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  130.622852] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  131.140837] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  131.658879] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  132.176882] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  132.694862] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  133.212867] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  133.730844] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  134.248857] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  134.766877] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  135.284865] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  135.802862] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  136.320859] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  136.838857] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  137.356866] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  137.874872] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  138.392843] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  138.910852] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  139.428873] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  139.944862] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  140.462863] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  140.980853] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  141.498875] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  142.016874] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  142.534871] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  143.052856] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  143.570866] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  144.088857] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  144.606848] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  145.124849] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  145.642853] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  146.160905] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  146.678866] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  147.196857] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  147.714864] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  148.232879] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  148.750860] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  149.268874] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  149.786869] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  150.304880] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  150.822861] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  151.340841] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  151.858844] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  152.376832] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  152.895854] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  153.411875] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  153.929854] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  154.447881] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  154.965858] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  155.483930] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  156.001891] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  156.519912] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  157.037835] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  157.555840] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  158.073844] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  158.591842] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  159.109846] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  159.627858] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  160.145874] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  160.663856] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  161.181872] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  161.699881] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  162.217858] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  162.735871] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  163.253898] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  163.771859] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  164.289879] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  164.807939] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  165.325853] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  165.843849] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  166.361874] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  166.879865] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  167.397870] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  167.915853] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  168.433842] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  168.951856] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  169.469886] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  169.987872] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  170.505843] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  171.023857] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  171.541872] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  172.059832] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  172.577853] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  173.095830] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  173.613847] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
[  174.129871] dracut-initqueue[395]: Warning: dracut-initqueue timeout - starting timeout scripts
```

**Comment by syuu1228** [comment]
@hagitsegev @yaronkaikov Do you see this issue on Jenkins AMI builds?
It only CentOS7 issue but can happen any versions of Scylla AMI builds.

**Comment by syuu1228** [comment]
I guessed it can be fix by re-generating initramfs, so I tried
```
run('dracut -f /boot/initramfs-{mlkver}.img {mlkver}'.format(mlkver=mlkver))
```
but still same error happens.

**Comment by syuu1228** [comment]
@avikivity do you have any idea?

**Comment by syuu1228** [comment]
New version of kernel-ml package released (5.14.1-1.el7.elrepo), so I tested it on CentOS official AMI again,
but same result:
```
[[32m  OK  [0m] Mounted Configuration File System.
[    3.190677] FDC 0 is a S82078B
[    3.237775] scsi host0: ata_piix
[    3.244619] xen_netfront: Initialising Xen virtual ethernet driver
[    3.269750] scsi host1: ata_piix
[    3.284906] ata1: PATA max MWDMA2 cmd 0x1f0 ctl 0x3f6 bmdma 0xc100 irq 14
[    3.292348] ata2: PATA max MWDMA2 cmd 0x170 ctl 0x376 bmdma 0xc108 irq 15
[  128.115617] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  128.647350] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  129.169369] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  129.692301] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  130.215406] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  130.739369] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  131.263336] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  131.787392] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  132.311339] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  132.835369] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
[  133.355367] dracut-initqueue[290]: Warning: dracut-initqueue timeout - starting timeout scripts
```
